org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/groovy/SmartApp.groovy

```diff
@@ -358,17 +354,15 @@ private getDeviceType(device) {
 			case "video camera":
 				deviceType = "camera"
                 return deviceType
-			case "motion sensor":
-				deviceType = "motionSensor"
-                return deviceType
-			case "presence sensor":
-				deviceType = "presenceSensor"
-                return deviceType
 			case "thermostat":
 				deviceType = "thermostat"
-                return deviceType
+                return deviceType    
+            case "acceleration sensor":
+			case "contact sensor":
+            case "motion sensor":
+			case "presence sensor":
```

Comment:
Nit:  lots of mixed tabs and spaces in this file, can you tabify or untabify the document.